.github/workflows/php-cs-fixer.yml

```diff
@@ -0,0 +1,23 @@
+name: Check & fix styling
```

Comment:
Static linting and tools like php-cs are fantastic and I'm a big fan. However, automatic code commits scare me, especially when there are things like "--allow-risky=yes". If that project gets compromised, or introduces changes it will then automatically commit those back to this project without a human reviewing it. 

PHP CS = good, automatic commits will not be accepted though. If you're willing to drop the automatic fix then we can just error on php cs issues.